linera-views/src/indexed_db.rs

```diff
@@ -46,11 +53,13 @@ const DATABASE_NAME: &str = "linera";
 /// API](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API#:~:text=IndexedDB%20is%20a%20low%2Dlevel,larger%20amounts%20of%20structured%20data.).
 pub struct IndexedDbStore {
     /// The database used for storing the data.
-    pub database: IdbDatabase,
+    pub database: Arc<IdbDatabase>,
     /// The object store name used for storing the data.
     pub object_store_name: String,
     /// The maximum number of queries used for the stream.
     pub max_stream_queries: usize,
+    /// The used root_key
```

Comment:
```suggestion
    /// The used root key
```

(Or "key of the root view"?)

Reply:
Ok.